securitycenter/notifications/update_notification_config.go

```diff
@@ -39,14 +39,16 @@ func updateNotificationConfig(w io.Writer, orgID string, notificationConfigID st
 	defer client.Close()
 
 	updatedDescription := "Updated sample config"
+	updatedFilter := `state = \"INACTIVE\"`
 	req := &securitycenterpb.UpdateNotificationConfigRequest{
 		NotificationConfig: &securitycenterpb.NotificationConfig{
 			Name:        fmt.Sprintf("organizations/%s/notificationConfigs/%s", orgID, notificationConfigID),
 			Description: updatedDescription,
 			PubsubTopic: updatedPubsubTopic,
+			StreamingConfig: &securitycenterpb.StreamingConfig{Filter: updatedFilter},
```

Comment:
Looks like this field isn't in the generated `NotificationConfig` type yet. When was it released? We regenerate the client every day. That's currently causing a compilation failure for this change. So, we won't be able to merge this until the generated type is updated.

See https://pkg.go.dev/google.golang.org/genproto/googleapis/cloud/securitycenter/v1?tab=doc#NotificationConfig.